samples/snippets/src/main/java/aiplatform/CreateFeatureSample.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ *
+ * Create a single feature for an existing entity type. See
+ * https://cloud.google.com/vertex-ai/docs/featurestore/setup before running 
+ * the code snippet
+ */
+
+package aiplatform;
+
+// [START aiplatform_create_feature_sample]
+
+import com.google.api.gax.longrunning.OperationFuture;
+import com.google.cloud.aiplatform.v1.CreateFeatureOperationMetadata;
+import com.google.cloud.aiplatform.v1.CreateFeatureRequest;
+import com.google.cloud.aiplatform.v1.EntityTypeName;
+import com.google.cloud.aiplatform.v1.Feature;
+import com.google.cloud.aiplatform.v1.Feature.ValueType;
+import com.google.cloud.aiplatform.v1.FeaturestoreServiceClient;
+import com.google.cloud.aiplatform.v1.FeaturestoreServiceSettings;
+import java.io.IOException;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+public class CreateFeatureSample {
+
+  public static void main(String[] args)
+      throws IOException, InterruptedException, ExecutionException, TimeoutException {
+    // TODO(developer): Replace these variables before running the sample.
+    String project = "YOUR_PROJECT_ID";
+    String featurestoreId = "YOUR_FEATURESTORE_ID";
+    String entityTypeId = "YOUR_ENTITY_TYPE_ID";
+    String featureId = "YOUR_FEATURE_ID";
+    String description = "YOUR_FEATURE_DESCRIPTION";
+    String valueType = "YOUR_FEATURE_VALUE_TYPE";
+    String location = "us-central1";
+    String endpoint = "us-central1-aiplatform.googleapis.com:443";
+    int timeout = 300;
```

Comment:
Should we follow the same timeout strategy as suggested in https://github.com/googleapis/nodejs-ai-platform/pull/338

Reply:
That was suggested for Featurestore API's, should follow the same timeouts as below for EntityType and Feature API's as well ?
15min for create
5min for delete
10min for update
5sec for list,get

Reply:
Lets do this instead, as Update on Feature/EntityType is syncronous, no long running operation i.e. 

15min for create
5min for delete
1min for update
5sec for list,get